scripts/build-images.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+# shellcheck disable=SC2002
```

Comment:
I would suggest we add SC2002 to `.shellcheckrc` file in eunomia instead of disabling it explicitly in scripts